+Æ’aÂµSâ€  v1.9.0-+-------------------------------------------------+
| id          | analytics                                       |
| transport   | [URL('kafka://localhost:9092')]                 |
| store       | memory:                                         |
| web         | http://localhost:6066/                          |
| log         | -stderr- (warn)                                 |
| pid         | 7325                                            |
| hostname    | pascal-asus                                     |
| platform    | CPython 3.7.3 (Linux x86_64)                    |
|        +    | Cython (GCC 8.3.0)                              |
| drivers     |                                                 |
|   transport | aiokafka=1.1.3                                  |
|   web       | aiohttp=3.5.4                                   |
| datadir     | /home/hadoop/kafka_2.11-2.3.0/analytics-data    |
| appdir      | /home/hadoop/kafka_2.11-2.3.0/analytics-data/v1 |
+-------------+-------------------------------------------------+
[?25l[?25h OK ^
[2019-11-01 00:55:44,090] [7325] [ERROR] [^-App]: Crashed reason=TypeError('Object of type PageVisit is not JSON serializable') 
Traceback (most recent call last):
  File "/home/hadoop/.local/lib/python3.7/site-packages/mode/services.py", line 770, in _execute_task
    await task
  File "/usr/local/lib/python3.7/dist-packages/faust-1.9.0-py3.7-linux-x86_64.egg/faust/app/base.py", line 922, in _wrapped
    return await task()
  File "/usr/local/lib/python3.7/dist-packages/faust-1.9.0-py3.7-linux-x86_64.egg/faust/app/base.py", line 971, in around_timer
    await fun(*args)
  File "/home/hadoop/kafka_2.11-2.3.0/analytics.py", line 98, in PageVisitSpout
    value=json.dumps( visit ).encode(),
  File "/usr/lib/python3.7/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type PageVisit is not JSON serializable
[2019-11-01 00:55:46,096] [7325] [CRITICAL] [^Worker]: We experienced a crash! Reraising original exception... 
[?25h